docker-compose.yml

```diff
@@ -10,3 +10,23 @@ services:
       - .:/var/www/
     ports:
       - "8888:80"
+    depends_on: 
+      - composer
+  composer:
+    image: composer:2.1
+    command: ["composer", "install"]
+    volumes:
+      - .:/var/www/
+    working_dir: /var/www/
+  npm:
+    image: node:16-alpine3.12
+    container_name: npm
+    volumes:
+      - .:/var/www/
+    command: sh -c "npm install --quiet &&
+                    cp node_modules/datatables.net-bs5/css/dataTables.bootstrap5.min.css /var/www/public_html/assets/css &&
+                    cp node_modules/datatables.net-bs5/js/dataTables.bootstrap5.min.js /var/www/public_html/assets/js &&
+                    cp node_modules/bootstrap/dist/js/bootstrap.bundle.min.* /var/www/public_html/assets/js &&
+                    cp node_modules/jquery/dist/jquery.min.* /var/www/public_html/assets/js/ &&
```

Comment:
Could we add these into the `npm` `package.json` `script` commands instead? Might be cleaner to have it all together..